docs/_edot-sdks/features.yml

```diff
@@ -158,8 +158,8 @@ features:
       status: ga
       min_version: 
     Node.js:
-      status: ga
-      min_version: 
+      status: tech-preview
```

Comment:
review note: The Logs API, Logs SDK, and instrumentations of logging libraries are still experimental in upstream OTel JS, so I was hesitant to call this "GA" in EDOT Node.js. Same for the log correlation below.